week-3/C-array-every/exercise.js

```diff
@@ -5,9 +5,14 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = students.every(function(element) {
+  for (i = 0; i < students.length; i++) {
+    return element;
+  }
+}) // complete this statement
 
-if (groupIsOnlyStudents) {
+
+if (groupIsOnlyStudents == students) {
```

Comment:
Hmm this doesn't make sense. groupIsOnlyStudents is a result of the `every()` function, so will be a boolean value (true or false). That shouldn't be compared to students, which is an array!